package.json

```diff
@@ -65,56 +65,58 @@
     ]
   },
   "dependencies": {
-    "@angular/animations": "15.0.0",
-    "@angular/cdk": "14.2.2",
-    "@angular/common": "15.0.0",
-    "@angular/compiler": "15.0.0",
-    "@angular/core": "15.0.0",
-    "@angular/forms": "15.0.0",
-    "@angular/material": "14.2.2",
-    "@angular/platform-browser": "15.0.0",
-    "@angular/platform-browser-dynamic": "15.0.0",
-    "@angular/platform-server": "15.0.0",
-    "@angular/router": "15.0.0",
-    "@nrwl/angular": "15.1.1",
+    "@angular/animations": "15.1.5",
+    "@angular/cdk": "15.1.6",
+    "@angular/common": "15.1.5",
+    "@angular/compiler": "15.1.5",
+    "@angular/core": "15.1.5",
```

Comment:
An update to Angular v15.2 is currently not possible because of the `jest-preset-angular` bug with snapshot tests. For more info, see: https://github.com/thymikee/jest-preset-angular/issues/2003

Reply:
Thanks @rubiesonthesky! I'll check.